planner/core/planbuilder.go

```diff
@@ -217,10 +217,17 @@ func (b *PlanBuilder) Build(node ast.Node) (Plan, error) {
 		return b.buildSimple(node.(ast.StmtNode))
 	case ast.DDLNode:
 		return b.buildDDL(x)
+	case *ast.ChangeStmt:
+		return b.buildChange(x)
 	}
 	return nil, ErrUnsupportedType.GenWithStack("Unsupported type %T", node)
 }
 
+func (b *PlanBuilder) buildChange(v *ast.ChangeStmt) (Plan, error) {
+	exe := &Change{Statement: v}
+	return exe, nil
```

Comment:
how about:
```go
return &Change{Statement: v}, nil
```

Reply:
planner/core/planbuilder.go:177   in this line ,we defined a method 'Build' , and it need return 'Plan'/'error' ,so this i can't change.

Reply:
finished